test/cypress/e2e/registration.spec.js

```diff
@@ -0,0 +1,118 @@
+import networks from '../../constants/networks';
+
+const ss = {
+  networkDropdown: '.network',
+  networkStatus: '.network-status',
+  newAccountBtn: '.new-account-button',
+  nodeAddress: '.peer',
+  errorPopup: '.toast',
+};
+
+/**
+ * Generates a sequence of random pairs of x,y coordinates on the screen that simulates
+ * the movement of mouse to produce a pass phrase.
+ */
+const moveMouseRandomly = () => {
+  for (let i = 0; i < 70; i += 1) {
+    cy.get('main').first().trigger('mousemove', {
+      which: 1,
+      pageX: 500 + (Math.floor((((i % 2) * 2) - 1) * (249 + (Math.random() * 250)))),
+      pageY: 500 + (Math.floor((((i % 2) * 2) - 1) * (249 + (Math.random() * 250)))),
+    });
+  }
+};
+
+const chooseNetwork = (network) => {
+  switch (network) {
+    case 'main':
+      cy.get(ss.networkDropdown).click();
+      cy.get('ul li').eq(1).click();
+      break;
+    case 'test':
+      cy.get(ss.networkDropdown).click();
+      cy.get('ul li').eq(2).click();
+      break;
+    case 'dev':
+      cy.get(ss.networkDropdown).click();
+      cy.get('ul li').eq(3).click();
+      cy.get('.address input').type(networks.devnet.node);
+      break;
+    case 'invalid':
+      cy.get(ss.networkDropdown).click();
+      cy.get('ul li').eq(3).click();
+      cy.get('.address input').type('http://silk.road');
+      break;
+    default:
+      throw new Error(`Network should be one of : main , test, dev, invalid . Was: ${network}`);
+  }
+};
+
+const registerUI = function () {
+  moveMouseRandomly();
+  cy.get('.get-passphrase-button').click();
+  cy.get('.i-understand-checkbox').click();
+  cy.get('.reveal-checkbox')
+    .trigger('mousedown')
+    .trigger('mousemove', { which: 1, pageX: 100, pageY: 0 })
+    .trigger('mouseup');
+  cy.get('textarea.passphrase').invoke('text').as('passphrase');
+  cy.get('.yes-its-safe-button').click();
+  cy.get('.passphrase-holder label').each(($el) => {
+    if (this.passphrase.includes($el[0].textContent)) cy.wrap($el).click();
+  });
+  cy.get('.get-to-your-dashboard-button').click();
```

Comment:
Is there a reason to use the selectors in `registerUI`, like `.get-to-your-dashboard-button` and not to define them in `ss` object like other selectors.

Reply:
The reason is I don't see what to do better yet. When I need to reuse selectors.
1) To move all selectors inside a special file. So we could import and use it in dev and qa bidirectionally.
2) To import some selectors that we need from one spec file from the other spec file.

Depending on the decision this batch of selectors are awaiting their fate cause they are bulky and only will be used inside this function.
Any thoughts?